### PR TITLE
aws-codecommit: fix Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kant** [comment]
[CI](https://github.com/tldr-pages/tldr/actions/workflows/ci.yml)

File: pages.es/common/aws-codecommit.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# aws codecommit
+
+> AWS CodeCommit es un servicio de control de origen administrado que aloja repositorios Git privados.
+> Más información: <https://docs.aws.amazon.com/cli/latest/reference/codecommit/>.
+
+- Muestra ayuda para un comando específico:
+
+`aws codecommit {{comando}} help`
+
+- Muestra ayuda:
```
Comment (navarroaxel) [inline pages.es/common/aws-codecommit.md:10]:
```suggestion
- Muestra la ayuda:
```

File: pages.es/common/aws-codecommit.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# aws codecommit
+
+> AWS CodeCommit es un servicio de control de origen administrado que aloja repositorios Git privados.
+> Más información: <https://docs.aws.amazon.com/cli/latest/reference/codecommit/>.
+
+- Muestra ayuda para un comando específico:
```
Comment (navarroaxel) [inline pages.es/common/aws-codecommit.md:6]:
```suggestion
- Muestra la ayuda para un comando específico:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**